lib/component.js

```diff
@@ -652,6 +676,68 @@ class Component extends WebComponent {
       }
     }
   }
+
+  _findNearestPanelAncestor() {
+    if (!this.isConnected) {
+      throw new Error(`Cannot determine context before component is connected to the DOM`);
+    }
+
+    let node = this.parentNode;
+    while (node) {
+      if (node._getActiveContexts) {
+        return node;
+      }
+      if (node.constructor.name === `ShadowRoot`) {
+        node = node.host;
+      } else {
+        node = node.parentNode;
+      }
+    }
+    return null;
+  }
+
+  _findAndMergeActiveContexts() {
+    const contextParent = this._findNearestPanelAncestor();
+    const defaultContexts = this.getConfig(`defaultContexts`);
+
+    if (contextParent) {
+      const parentContexts = contextParent._getActiveContexts();
+
+      if (parentContexts.size > 0) {
+        // merge context maps
+        const combinedContexts = new Map(parentContexts.entries());
```

Comment:
Yes, that's accurate. Contexts must always declare a default, because the component should function even if a production-ready Context is not provided by ancestors.